public/video-ui/src/services/pandaReqwest.js

```diff
@@ -37,10 +37,11 @@ function poll(reqwestBody, timeout) {
 export function pandaReqwest(reqwestBody, timeout = 0) {
   const payload = Object.assign({ method: 'get' }, reqwestBody);
 
-  if(payload.data) {
+  if (payload.data) {
     payload.contentType = payload.contentType || 'application/json';
 
-    if(payload.contentType === 'application/json' && typeof payload.data === 'object') {
+    // prettier-ignore
```

Comment:
otherwise prettier goes crazy and turns this into

```js
if (
    payload.contentType === 'application/json' &&
    typeof payload.data === 'object'
) {
    payload.data = JSON.stringify(payload.data);
}
```